code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -117,40 +113,32 @@ rcd light flash thingy on matter drain
 	if(C)
 		C.post_status("alert", "lockdown")
 
-	src.verbs += /mob/living/silicon/ai/proc/disablelockdown
-	src << "<span class = 'warning'>Lockdown Initiated.</span>"
-	malf_cooldown = 1
-	spawn(30)
-	malf_cooldown = 0
+	verbs -= /mob/living/silicon/ai/proc/lockdown
+	minor_announcement.Announce("Hostile runtime detected in door controllers. Isolation lockdown protocols are now in effect. Please remain calm.","Network Alert")
+	src << "<span class='danger'>Lockdown initiated. Network reset in 90 seconds.</span>"
+	spawn(900) //90 Seconds.
+		disablelockdown() //Reset the lockdown after 90 seconds.

```

Comment:
I believe it will runtime because it's a mob proc (see below). I could be terribly wrong about this, though!